src/htmx.js

```diff
@@ -1236,26 +1236,33 @@ var htmx = (function() {
    * @property {EventTarget} target
    * @property {AnyEventName} event
    * @property {EventListener} listener
+   * @property {Object|boolean} options
    */
 
   /**
    * @param {EventTarget|AnyEventName} arg1
    * @param {AnyEventName|EventListener} arg2
-   * @param {EventListener} [arg3]
+   * @param {EventListener|Object|boolean} [arg3]
+   * @param {Object|boolean} [arg4]
    * @returns {EventArgs}
    */
-  function processEventArgs(arg1, arg2, arg3) {
+  function processEventArgs(arg1, arg2, arg3, arg4) {
+    if (arg4 == null) {
+      arg4 = {}
```

Comment:
Is this necessary? Doesn't addEventListener gracefully handle a null (as well as undefined) value?

Reply:
You’re right, it does! Removed that redundant code, thanks.